test/units/testsuite-04.sh

```diff
@@ -290,4 +290,36 @@ rm -rf "$JTMP"
 
 rm /tmp/lb1
 
+# https://bugzilla.redhat.com/show_bug.cgi?id=2183546
+mkdir /run/systemd/system/systemd-journald.service.d
+MID=$(cat /etc/machine-id)
+for c in "NONE" "XZ" "LZ4" "ZSTD"; do
+    cat >/run/systemd/system/systemd-journald.service.d/compress.conf <<EOF
+[Service]
+Environment=SYSTEMD_JOURNAL_COMPRESS=${c}
+EOF
+    systemctl daemon-reload
+    systemctl restart systemd-journald.service
+    journalctl --rotate
+
+    ID=$(systemd-id128 new)
+    systemd-cat -t "$ID" /bin/sh -c "for ((i=0;i<100;i++)); do echo -n hoge with ${c}; done; echo"
```

Comment:
`for i in {1..100}; do …`

The logs in ubuntuautopkgtests fail with:
```
Apr 04 20:37:26 H 6c9593a7ec374bb095fba079bf38e79f[844]: /bin/sh: 1: Syntax error: Bad for loop variable
```
I don't know if this this line, but it seems the most likely. But we'd want the normal syntax anyway.

Reply:
Changed to /bin/sh -> /bin/bash. Hopefully now it works.